src/include/nodes/nodeFuncs.h

```diff
@@ -14,6 +14,7 @@
 #define NODEFUNCS_H
 
 #include "nodes/parsenodes.h"
+#include "nodes/plannodes.h"
```

Comment:
why?

Reply:
First argument of `pltsql_exprTypmod_hook_type` is of type Plan, and Plan struct is defined in plannodes.h